crates/hir-def/src/import_map.rs

```diff
@@ -320,7 +320,7 @@ impl SearchMode {
                     };
                     match m {
                         Some((index, _)) => {
-                            name = &name[index + 1..];
+                            name = &name[ceil_char_boundary(name, index + 1)..];
```

Comment:
Given we want to discard the string up to (and including, hence the + 1) the `query_char`, I think we can just do
```rs
let mut chars = name[index..].chars();
chars.next();
chars.as_str()
```
instead?

Reply:
Or
```rs
name = name[index..].strip_prefix(|_: char| true).unwrap_or_default()
```